### DISCUSSION TITLE
is there a way to adjust the size of the pipeline boxes?

### DISCUSSION THREAD

**Discussion by alex1x** [concourse/concourse#7028] [Help & Support]
Hi,

Is there a way to adjust the height and width of pipeline boxes? It looks like they double in size per maybe 12 blocks? e.g. the `main` pipeline block here https://medium.com/concourse-ci/concourse-update-jul-9-13-484c7fe922ec

It would be good to be able to override their size.

Thanks,
Alex

**Comment by taylorsilva** [comment]
Hey, there is no way to change the size of the boxes manually. The UI rendering isn't configurable at all. Haven't seen any requests for this recently. Are you ending up with some ugly looking pipelines?

**Reply by alex1x** [reply]
Nothing terrible, just a bit hard to see what is happening, and since the pipeline on the right is taking up the height real estate already, we would prefer the 2 pipelines on the left to also extend lower.

<img width="1293" alt="Screenshot 2021-05-18 at 17 20 04" src="https://user-images.githubusercontent.com/6977670/118687890-6ee45a00-b7fd-11eb-824e-3af6efb6579d.png">
